Packs/Whois/ReleaseNotes/1_5_16.json

```
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": "Commands <code>domain</code> and <code>whois</code> will now return a dictionary instead of a string when 'Use legacy context' is not enabled. "
+}
```

```suggestion
    "breakingChangesNotes": "Commands ***domain*** and ***whois*** will now return a dictionary instead of a string when 'Use legacy context' is disabled. "
}
```
 Also this is true only for raw response so let's make sure it is clear from the note.